LCM/dsc/engine/ca/CAInfrastructure/CAEngine.c

```diff
@@ -1286,10 +1286,11 @@ MI_Result MoveToDesiredState(_In_ ProviderCallbackContext *provContext,
     if (
 #if defined(BUILD_OMS)
         g_DscHost == MI_FALSE ||
+        Tcscasecmp(instance->classDecl->name, METACONFIG_CLASSNAME) == 0 || // put special cases to wmiv2 code
 #else
+        Tcscasecmp(instance->classDecl->name, METACONFIG_CLASSNAME) == 0 || // put special cases to wmiv2 code
```

Comment:
This was done to make sure dsc doesn't crase n case where this is true but reginstance isn't defined/